skyportal/handlers/api/internal/annotations_info.py

```diff
@@ -0,0 +1,143 @@
+from collections import defaultdict
+from sqlalchemy import func, literal
+from sqlalchemy.ext.compiler import compiles
+from sqlalchemy.sql import functions
+from sqlalchemy.sql.elements import ColumnClause
+from sqlalchemy.sql.selectable import FromClause
+from baselayer.app.access import auth_or_token
+from ...base import BaseHandler
+from ....models import (
+    DBSession,
+    Obj,
+    Candidate,
+    Annotation,
+    GroupAnnotation,
+)
+
+
+# Helper classes/functions below provide a workaround for PostgreSQL functions
+# returning multiple columns, unsupported by SA
+# See https://stackoverflow.com/questions/33865038/joining-with-set-returning-function-srf-and-access-columns-in-sqlalchemy
+# for details
+class FunctionColumn(ColumnClause):
+    def __init__(self, function, name, type_=None):
+        self.function = self.table = function
+        self.name = self.key = name
+        self.key = self.name
+        self.type_ = type_
+        self.is_literal = False
+
+    @property
+    def _from_objects(self):
+        return []
+
+    def _make_proxy(
+        self, selectable, name=None, attach=True, name_is_truncatable=False, **kw
+    ):
+        print('_make_proxy')
```

Comment:
I know this is just a copy/pasted recipe, but do we want to keep this print statement in here?